resources/lang/de.json

```
@@ -559,6 +559,11 @@
     "report-bug": "Fehler melden",
     "to-support": "Zum Support",
     "request-feature": "Funktion wünschen",
+    "about.events": "Was sind Veranstaltungen?",
+    "about.events.description1": "Träwelling bringt Menschen zusammen, die gemeinsam öffentliche Verkehrsmittel nutzen. Manchmal fahren sie sogar zu derselben Veranstaltung, ohne voneinander zu wissen!",
+    "about.events.description2": "Deswegen haben wir das Veranstaltungen-Feature integriert. Jede*r kann <a href=\":link\">über unser Formular</a> eine Veranstaltung für einen bestimmten Zeitraum anlegen und alle Benutzer*innen von Träwelling können ihre Check-ins dann mit dem Event verknüpfen.",
+    "about.events.description3": "Wenn du ein Event anlegen möchtest, beachte bitte, dass Veranstaltungen einen Mehrwert für die gesamte Community von Träwelling haben sollen. Dazu gehören Events aus dem Eisenbahnwesen wie bspw. <a href=\"https://traewelling.de/statuses/event/letzte_fahrt_des_met\">die letzte Fahrt des Metronoms</a>, Events von Hackerspaces (z.B. die <a href=\"https://traewelling.de/statuses/event/gpn20\">GPN</a>), oder aus der LGBTQ-Szene wie die <a href=\"https://traewelling.de/statuses/event/csd_koln_2022\">Cologne Pride</a>.",
```

oh.. thats what you mean...

met != metronom
met = metropolitan
```suggestion
    "about.events.description3": "Wenn du ein Event anlegen möchtest, beachte bitte, dass Veranstaltungen einen Mehrwert für die gesamte Community von Träwelling haben sollen. Dazu gehören Events aus dem Eisenbahnwesen wie bspw. <a href=\"https://traewelling.de/statuses/event/letzte_fahrt_des_met\">die letzte Fahrt des Metropolitan</a>, Events von Hackerspaces (z.B. die <a href=\"https://traewelling.de/statuses/event/gpn20\">GPN</a>), oder aus der LGBTQ-Szene wie die <a href=\"https://traewelling.de/statuses/event/csd_koln_2022\">Cologne Pride</a>.",
```